charts/tempo-distributed/values.yaml

```diff
@@ -0,0 +1,125 @@
+# -- Overrides the chart's computed fullname
+fullnameOverride: tempo
+# -- Overrides the chart's computed fullname
+tempo:
+  image:
+    # -- Docker image repository
+    repository: grafana/tempo
+    # -- Overrides the image tag whose default is the chart's appVersion
+    tag: latest
+    # -- Docker image pull policy
+    pullPolicy: IfNotPresent
+  distributor:
+    replicas: 2
```

Comment:
Make the default value of replicas 1, users should be able to decide how to scale Tempo.